tests/roles/standard-test-docker/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Install the docker requirements
+  package: name="{{item}}" state=latest
+  with_items:
+    - docker
+
+- name: Make artifacts directory
+  file: path={{ artifacts }} state=directory owner=root mode=755 recurse=yes
+
+- name: Start the docker service
+  service:
+    name: docker
+    state: running
+
+- name: Start ansible-ready docker container running SSHD
+  shell: docker run -d -p "{{docker_ssh_port}}:22" -v "{{artifacts}}:/artifacts:z,rw" -v "{{role_path}}/files/docker-run-ssh:/run.sh" "{{subjects}}" /bin/sh -ex /run.sh
+  register: docker_run_output
+
+- name: Capture docker container ID
+  set_fact: container_id="{{docker_run_output.stdout}}"
+
+- name: Wait for container to initialize
+  wait_for: port="{{docker_ssh_port}}" search_regex=OpenSSH
+
+- name: How to do this properly?
+  debug: msg="Need to run playbook {{playbooks}} in container {{container_id}} with SSH on port {{docker_ssh_port}} using inventory file {{role_path}}/files/docker-inventory"
+
```

Comment:
I need some assistance figuring out the proper way to run the nested playbook for the actual test run. See my comments in the code.